components/workflow-core/src/main/java/org/workflow/core/dboperation/DatabaseHandler.java

```diff
@@ -102,9 +102,13 @@ public HistoryResponse getApprovalHistory(String subscriber, String applicationN
                 .append("WHERE EXISTS( SELECT 1 FROM " + depDB + "." + Tables.DEP_OPERATOR_APPS.getTObject() + " opcoApp ")
                 .append("INNER JOIN " + depDB + "." + Tables.DEP_OPERATORS.getTObject() + " opco ON opcoApp.operatorid = opco.id ")
                 .append("WHERE opcoApp.isactive LIKE ? AND opcoApp.applicationid = amapp.application_id AND ")
-                .append("opco.operatorname LIKE ? AND amapp.application_id LIKE ? AND amapp.name LIKE ? AND amapp.subscriber_id LIKE ? ) ")
-                .append("AND amapp.application_status LIKE ? ")
-                .append("ORDER BY application_id) t")
+                .append("opco.operatorname LIKE ? AND amapp.application_id LIKE ? AND amapp.name LIKE ? AND amapp.subscriber_id LIKE ? ) ");
+
+            if(status!=null && !status.isEmpty()) {
```

Comment:
this should be equal to the cheking of parameter set process like
status!=null && !status.isEmpty() && status.equals(ALL)